bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/filter.xml

```diff
@@ -14,17 +14,27 @@
 		<channels>
 			<channel id="filterEnable" typeId="system.power"/>
 			<channel id="filterValvePosition" typeId="valvePosition"/>
-			<channel id="filterSpeed" typeId="filterSpeed"/>
+			<channel id="filterSpeedPercent" typeId="filterSpeedPercent"/>
+			<channel id="filterSpeedRpm" typeId="filterSpeedRpm"/>
+			<channel id="filterSpeedSelect" typeId="filterSpeedSelect"/>
```

Comment:
This could also be mentioned in the openHAB update messages.

Reply:
Is there any documentation available on the update.lst syntax?